src/supportpsx/binlua.cc

```
@@ -86,6 +87,12 @@ uint32_t uclWrapper(const uint8_t* in, uint32_t size, uint8_t* out) {
     return r == UCL_E_OK ? outSize : 0;
 }
 
+uint32_t uclGetOverlapMargin(const uint8_t* src, size_t srcLen, size_t expectedDstLen) {
```

Cppcheck (2.21.0)

[error] If memory allocation fails

(nullPointerArithmeticOutOfMemory)